CMS-Enterprise/easi-app
# [EASI-4591] IT Gov admin route updates

# EASI-4591

## Description
- Updated IT Gov admin route from "governance-review-team" to "it-governance"
  - Both admins and GRB reviewers now use the same route
  - Added reroute to handle old "governance-review-team" links
- Updated context object for determining GRB Reviewer views
  - `IsGrbViewContext` -> `ITGovAdminContext`
  - Updated conditional logic to show action buttons only when user is admin
- Unit tests 

## How to test this change
- Test IT Gov admin section as both admin and GRB reviewer
  - All tabs should load with "it-governance" route
  - Action buttons should NOT render for GRB reviewer
- Check that "/governance-review-team/:systemId/grb-review" reroutes to "/it-governance/:systemId/grb-review"

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
